### PR TITLE
regenerate with 0.16.1

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -55065,7 +54999,7 @@ pub mod sysctl {
                 self.w
             }
         }
-        #[doc = "Possible values of the field `dma_sel3`"]
+        #[doc = ""]
```
Comment (Disasm) [inline src/lib.rs:55002]: Maybe these empty doc attributes can be considered a regression in `svd2rust` or at least something to improve.

**Comment by Disasm** [comment]
Thanks!

File: src/lib.rs (hunk 1 above)
Comment (burrbull) [reply src/lib.rs:55002]: https://github.com/rust-embedded/svd2rust/pull/380
изображение

Comment (Disasm) [reply src/lib.rs:55002]: It's ok to not have any description here, but what is the reason for generating an empty `#[doc = ""]` attribute?

Comment (burrbull) [reply src/lib.rs:55002]: I discovered this issue at the same time as you.

**Pull request merged**